site/docs/reference/Connectors/capture-connectors/MySQL.md

```diff
@@ -0,0 +1,79 @@
+This is a change data capture (CDC) connector that captures change events from a MySQL database via the [Binary Log](https://dev.mysql.com/doc/refman/8.0/en/binary-log.html).
```

Comment:
Perhaps there should be a warning at the top level here that this connector is still under development and we make no guarantees about its readiness for production use?